README.md

```diff
@@ -9,6 +9,14 @@ The synchronization works like this
 * creating a new OpenShift Build for a BuildConfig associated with a Jenkins Job results in the Jenkins Job being triggered
 * changes in a Jenkins Build Run thats associated with a Jenkins Job gets replicated to an OpenShift Build object (which is created if necessary if the build was triggered via Jenkins)
 
+Configuration
+------------------------
+Jenkins Build Log URL: 
+* The plugin adds an annotation in Build resource metadata which contains URL to access Jenkins build logs.
+By default, the Jenkins base URL for the build log is determined via the OpenShift route of the jenkins service. To override and configure this base URL, you can set the environment variable `JENKINS_ROOT_URL`. 
+This environment variable will get higher precedence than Jenkins service to determine base URL.
```

Comment:
"higher precedence" is a misnomer. Precedence already implies the order and that it has a higher rank in this order.